lighthouse-core/lib/stack-packs-test/wordpress.json

```diff
@@ -0,0 +1,121 @@
+[
```

Comment:
copied from https://github.com/GoogleChrome/lighthouse-stack-packs/blob/master/wordpress/audits.json

Reply:
since this is a hacky fake package, lets just transform these descriptions to what we want the real stack pack definition to look like. (After we all approve this, we can do a quick PR on the stack-pack repo to match the defined format)

Might as well just use 1 file and move all these descriptions into the -package.js